Docs/ProjectSpec.md

```diff
@@ -447,6 +451,11 @@ A multiline script can be written using the various YAML multiline methods, for
 - [ ] **gatherCoverageData**: **Bool** - a boolean that indicates if this scheme should gather coverage data. This defaults to false
 - [ ] **targets**: **[String]** - a list of targets to test
 
+### Environment Variable
+- [x] **key**: **String** - variable's key.
```

Comment:
Maybe `variable` is a better name here. The yaml key can be changed here, but maybe also the property name in https://github.com/xcodeswift/xcproj/pull/227?
What do you think?

Reply:
I have updated that both in xcproj and XcodeGen! 💯